uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/absence-type-model.js

```diff
@@ -38,6 +40,26 @@ define([
          */
         calculateToilExpiryDate: function (absenceTypeID, date, params) {
           return absenceTypeAPI.calculateToilExpiryDate(absenceTypeID, date, params);
+        },
+
+        /**
+         * Determines if the absence type can expire by querying if
+         * the expiration unit and duration are null.
+         *
+         * @param   {string} absenceTypeId
+         * @return  {Promise}
+         */
+        canExpire: function (absenceTypeId) {
+          return absenceTypeAPI.all({
+            accrual_expiration_unit: { 'IS NULL': 1 },
+            accrual_expiration_duration: { 'IS NULL': 1 },
+            allow_accruals_request: 1,
+            id: absenceTypeId,
+            sequential: 1
```

Comment:
I don't think you need to specify `sequential: 1`. Please look at this file `org.civicrm.reqangular/src/common/services/api.js` which contains sendGet function. Better double-check, but I really think it is not needed.